### PR TITLE
Redid testing section

### PR DESCRIPTION
**Pull request by mhk69**

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -612,9 +612,33 @@ SenseSdkError | message | The error message
...
+<aside class="warning"> Forgetting to turn off a simulated location may result in non-accurate location on your device.
+</aside>
+
+## Debug Notifications
+
+If you wish to verify that your triggers and recipes are being registered properly, you can enable Debug Notifications with the SDK:
+
+1. Open your info.plist. (go to your project and hit the Info tab at the top of the screen)
+2. Add the key "sense360:sendDebugNotifications"
+3. Set it to a Boolean with a value of YES
```
Comment (KamilMroczek) [inline source/index.md]: Lets tell them what they will get.

You should get notifications every time we believe the phone has arrived at a place. At that point our 2nd tier logic starts to figure out exactly where the device is (i.e. a POI type, home, work, etc)

File: source/index.md [hunk 2]
```diff
@@ -765,12 +789,9 @@ For example, if you are testing entering a restaurant. Make sure that you are no
...
 
-1. Open your info.plist. (go to your project and hit the Info tab at the top of the screen)
-2. Add the key "sense360:sendDebugNotifications"
-3. Set it to a Boolean with a value of YES
-4. Make sure that at least one recipe is being registered (very important!)
-5. Turn your app on
-6. Go at least 1 kilometer away from your location, sit down, and wait for a notification!
+1. Make sure that at least one recipe is being registered (very important!)
+2. Turn your app on
+3. Go at least 1 kilometer away from your location, sit down, and wait for a notification!
```
Comment (KamilMroczek) [inline source/index.md]: Lets be a bit more clear here. For #3:

Move at least a kilometer from your current location and then go to a place that you expect your trigger to fire (e.g. a restaurant). Sit down and wait for a notification!

**Comment by KamilMroczek** [comment]
good stuff thats all i had

**Comment by mhk69** [comment]
updatedc

**Pull request merged**